client/my-sites/jetpack-search/jetpack-instant-search-config/behavioral.tsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement, Fragment } from 'react';
+import { translate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { Card } from '@automattic/components';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormLegend from 'calypso/components/forms/form-legend';
+import FormRadiosBar from 'calypso/components/forms/form-radios-bar';
+import MultiCheckbox from 'calypso/components/forms/multi-checkbox';
+import QueryPostTypes from 'calypso/components/data/query-post-types';
+import SettingsSectionHeader from 'calypso/my-sites/site-settings/settings-section-header';
+import { getPostTypes } from 'calypso/state/post-types/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+type PostType = {
+	public: boolean;
+	label: string;
+	name: string;
+};
+
+export default function JetpackSearchInstantSearchBehavioralConfig(): ReactElement {
+	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) as number );
+	const postTypes = useSelector(
+		( state ) => getPostTypes( state, siteId ) as Record< string, PostType >
+	);
+	const publicPostTypes: PostType[] =
+		typeof postTypes !== 'object'
+			? []
+			: Object.keys( postTypes )
+					.map( ( key ) => postTypes[ key ] )
+					.filter( ( postType ) => postType.public );
+	const checkboxOptions = publicPostTypes.map( ( postType ) => ( {
+		label: postType.label,
+		value: postType.name,
+	} ) );
+	const disableCheckboxes = false; // TODO: checked.length - 1 === checkboxOptions.length
+
+	return (
+		<Fragment>
+			{ siteId && <QueryPostTypes siteId={ siteId } /> }
+			<SettingsSectionHeader
+				id="jetpack-search-behavior-settings"
+				showButton
+				title={ translate( 'Behavioral', { context: 'Settings header' } ) }
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.